report/report_test.go

```diff
@@ -89,3 +91,21 @@ func TestPongo2Report(t *testing.T) {
 		t.Fatalf("error writing report\n%s", err)
 	}
 }
+
+func TestDistributionList(t *testing.T) {
+	df, err := ReadDistributionFormatYAMLFromFile("distributionListTest.yml")
+	if err != nil {
+		t.Fatalf("Failed to read distribution format\n%s", err)
+	}
+
+	// using reflection can work for the general use case
+	severityList := reflect.ValueOf(&df.Severity).Elem()
+	severityType := severityList.Type()
+	for i := 0; i < severityList.NumField(); i++ {
+		f := severityList.Field(i)
+		fmt.Printf("%d: %s %s = %v\n", i,
+			severityType.Field(i).Name, f.Type(), f.Interface())
+	}
+
+	df.Print()
```

Comment:
Can we print this as a Debug message, so it doesn't clutter the test results unnecessarily?